### PR TITLE
First draft of contributor doc.

### PR DESCRIPTION
**Pull request by acinader**
Proposal to establish a paid contributor program to promote useful contributions.

### CODE REVIEW

**Review by davimacedo — APPROVED** [review]

File: paid-contributors.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Parse Community Paid Contributors
+
+In an effort to maintain the quality and timely development of the parse platform, the parse-community will pay a monthly stipend to contributors who have demonstrated a consistent, positive contribution to the community.
```
Comment (TomWFox) [inline paid-contributors.md]:
```suggestion
In an effort to maintain the quality and timely development of the Parse Platform, the Parse Community will pay a monthly stipend to contributors who have demonstrated a consistent, positive contribution to the community.
```

File: paid-contributors.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+This program is a pilot program whose effectiveness will be evaluated to determine if it should be continued.
```
Comment (TomWFox) [inline paid-contributors.md]:
```suggestion
This is a pilot program whose effectiveness will be evaluated to determine if it should be continued.
```

File: paid-contributors.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+The goal is to compensate individual contributors for whom the payments will be a material incentive to enable them to continue their contributions.
+
+Initially, the core contributor group will determine by vote who will be eligible for a stipend.  A monthly amount will be agreed to and voted on by the core contributor group.  The number of core contributors receiving funds must be less than a majority of the members.
+
+Each month the recipient will submit a payment request to https://opencollective.com/parse-server for review and approval.  The payment request should include a brief summary of the months activities to make transparency easy for any observer. Payment request submitted without having been approved for the program will be denied.
```
Comment (TomWFox) [inline paid-contributors.md]:
```suggestion
Each month the recipient will submit a payment request to our [Open Collective](https://opencollective.com/parse-server) for review and approval.  The payment request should include a brief summary of the months activities to make transparency easy for any observer. Payment requests submitted without having been approved for the program will be denied.
```

**Review by TomWFox — APPROVED** [review]
Looks good, just some minor changes

**Comment by TomWFox** [comment]
@acinader Out of interest, what do you think of the suggested change thing?

**Comment by acinader** [comment]
first time i've seen.  i don't even know how you did that, but i'll figure out for my next one.  

AWESOME!

**Comment by TomWFox** [comment]
@acinader the vote has closed with no objections so I'd say we're good to merge this.

**Pull request merged**